backend-zs/models/project.model.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require('mongoose')
+require('mongoose-type-url')
+
+const Point = new mongoose.Schema({
+    type: {
+        type: String,
+        enum: ['Point'],
+        required: true
+    },
+    coordinates: {
+        type: [Number],
+        required: true
+    }
+});
+
+
+var Project = mongoose.Schema({
```

Comment:
The definition of a mongoose Schema requires instantiation with 'new'.

Reply:
Both work the same way (https://stackoverflow.com/questions/39719454/creating-mongoose-schemas-with-or-without-new-keyword). I wanted to be consistent with the implementation of the user model.

Reply:
In that case, I guess we should remove 'new' from the Point instantiation